modular_skyrat/modules/titlescreen/code/titlescreen_html.dm

```diff
@@ -0,0 +1,99 @@
+
+GLOBAL_LIST_EMPTY(startup_messages)
+// FOR MOR INFO ON HTML CUSTOMISATION, SEE: https://github.com/Skyrat-SS13/Skyrat-tg/pull/4783
+
+/mob/dead/new_player/proc/get_lobby_html()
+	var/dat = GLOB.lobby_html
+	if(SSticker.current_state == GAME_STATE_STARTUP)
+		dat += {"<img src="titlescreen.gif" class="fone" alt="">"}
+		dat += {"
+		<div class="container_terminal">
+			<p class="menu_b">SYSTEMS INITIALIZING:</p>
+		"}
+		var/loop_index = 0
+		for(var/i in GLOB.startup_messages)
+			if(loop_index >= 27)
```

Comment:
why 27? may want to define it as well

Reply:
It's how many can fit on screen comfortably